login-fire.html

```diff
@@ -84,25 +84,23 @@
     }
 
     .emailPWError {
-      color: #dd2c00;
-      direction: ltr;
+      color: var(--error-color);
```

Comment:
`--error-color` is defined here: https://github.com/PolymerElements/paper-styles/blob/master/default-theme.html